packages/table/src/table.tsx

```diff
@@ -177,6 +177,14 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     onCopy?: (success: boolean) => void;
 
     /**
+     * A callback called when the visible cell indices change in the table.
+     */
+    onVisibleCellsChange?: (rowStartIndex: number,
```

Comment:
This actually messes up the formatting by making the page overly wide. :/ What's the best workaround? Leave as is, or maybe:

```tsx
onVisibleCellsChange?: (rowIndices: IRowIndices, columnIndices: IColumnIndices) => void;`
```

?